jack-and-the-beans/buddies-ios
# Chat improvements

Adds avatars, names, system messages.
Fixes hiding keyboard issues.

To Test
- join various activities
- send messages
- join as a different user & send messages
- leave activities to see system messages

To Do
- [ ] Blocked users / users removed from the chat
- [ ] Feedback regarding visuals / whatever else.
